stage8/02-extra-boot/00-run.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash -e
 
 if [ -n ${EXTRA_BOOT} ]; then
-	wget -r -nH --cut-dirs=4 -np -R "index.html*" -l2 "${EXTRA_BOOT}" -P "${STAGE_WORK_DIR}/rootfs/boot"
+	wget -r -nH --cut-dirs=4 -np -R "index.html*" -l3 "${EXTRA_BOOT}" -P "${STAGE_WORK_DIR}/rootfs/boot"
```

Comment:
could we make this somehow parametrize this with some defaults?
if this changes again we should be able to configure this from the CI build;

Reply:
this is unlikely to change in the near future. To parametrize this would require a new build to be done. I would have it as a future improvement.

Reply:
you just do `EXTRA_BOOT_DIR_DEPTH=${EXTRA_BOOT_DIR_DEPTH:-3}`
and then  `wget -r -nH --cut-dirs=4 -np -R "index.html*" "-l${EXTRA_BOOT_DIR_DEPTH}" "${EXTRA_BOOT}" -P "${STAGE_WORK_DIR}/rootfs/boot"`

same for `--cut-dirs` if needed;